Dockerfile

```
@@ -38,17 +37,15 @@ WORKDIR /srv
 RUN chown -R node:node /srv/ && \
     chmod 0750 /srv/
 
-COPY --chown=node:node package.json yarn.lock /srv/
-RUN chmod 0640 /srv/package.json /srv/yarn.lock
+COPY --chown=node:node . /srv
 
 ################################################################
 
 USER node
-RUN yarn install --frozen-lockfile
+RUN yarn install --immutable
 
 ################################################################
 
```

```suggestion

COPY --chown=node:node . /srv/

```
We need to copy the source code now.